src/exchange-concrete.js

```diff
@@ -8,8 +8,9 @@ var Quote = require('./quote');
 var PaymentMedium = require('./payment-medium');
 
 class Exchange extends AbstractExchange {
```

Comment:
No, they are only used for tests, so I might move them to the tests folder at some point.